packages/perseus/src/widgets/matrix/matrix.tsx

```diff
@@ -537,16 +533,27 @@ class Matrix extends React.Component<Props, State> implements Widget {
     }
 }
 
-const propTransform: (arg1: any) => any = (editorProps) => {
+// NOTE: Answers here in RenderProps does not contain answers from the rubric.
+// It's an empty array of the same length and depth used to render the matrix.
```

Comment:
Let me know if this comment doesn't seem helpful. I always take a second look when I see the word "answers", so wanted to clarify.

Reply:
Yeah, I think we could rename this from `answers`. Without looking, I suspect it's also mutate with user input like Table is? Hm...but like Table has `rows`/`columns`, Matrix has `matrixBoardSize`. So I bet it's getting dimensions from one and storing user input in another?

Reply:
It's more along the lines of what @handeyeco describes. Answers is the box that holds the user's answers. It uses a function called `getMatrixSize` to iterate over the matrix and get the size based on which rows and cells actually contain data. I think this can just be changed to use `matrixBoardSize` since rows and cells are not supposed to be null/empty... but I'm not sure. I made the change and there is a snapshot change. I'll investigate a bit more tomorrow and also try renaming `answers` to `userInput` or something, at least in the places where it represents that. 🤔